src/openapi/lab.yml

```diff
@@ -386,13 +386,13 @@ endpoints:
           properties:
             name:
               type: string
-              pattern: '^[\w%]+$'
+              pattern: '^[\w\-%]+$'
```

Comment:
In region model, `name` and `location` is string w/o any pattern restrictions. My stupid mistake when I was adding the filter feature... You can remove `pattern` completely.